lib/test-worker.js

```diff
@@ -86,6 +86,12 @@ var oldNodeModulesPaths = module.constructor._nodeModulePaths;
 module.constructor._nodeModulePaths = function () {
 	var ret = oldNodeModulesPaths.apply(this, arguments);
 	ret.push(nodeModulesDir);
+	if (process.env.NODE_PATH) {
+		var osSplitChar = /^win/.test(process.platform) ? ';' : ',';
+		process.env.NODE_PATH.split(osSplitChar).forEach(function (additionalPath) {
+			ret.push(path.join(opts.projectRoot, additionalPath));
```

Comment:
Should use [`path.resolve`](https://nodejs.org/api/path.html#path_path_resolve_from_to) in case `additionalPath` is already absolute.